release/gcb/helm_charts.sh

```diff
@@ -55,15 +52,10 @@ done
 
 $HELM repo index "$HELM_BUILD_DIR"
 
-# Copy the new built helm repo to the target dir.
-mkdir -vp "$CHARTS_TARGET_DIR"
-
-cp -vr "${HELM_BUILD_DIR}/*" "$CHARTS_TARGET_DIR"
+# Copy output to GCS bucket.
+gsutil -qm cp -rP "${HELM_BUILD_DIR}/*" "gs://${CB_GCS_BUILD_PATH}/charts/"
```

Comment:
no you really need Posix flag? the following should be good enough
gsutil -qm cp -r "${HELM_BUILD_DIR}/*" "gs://${CB_GCS_BUILD_PATH}/charts/"